src/main/java/eu/europeana/metis/sandbox/service/workflow/HarvestServiceImpl.java

```diff
@@ -331,4 +332,9 @@ private RecordInfo handleDuplicated(String providerId, Step step, Record.RecordB
     saveErrorWhileHarvesting(recordToHarvest, providerId, step, new RuntimeException(recordErrorCreated.getMessage()));
     return null;
   }
+
+  private String createTemporaryIdFromPath(String extractedDirectory, Path pathToRelativize){
+    return !extractedDirectory.isBlank() ? Paths.get(extractedDirectory).relativize(pathToRelativize).toString() :
```

Comment:
Suggestion to change the `if` to a positive expression to make it easy to read :wink: 
```
return extractedDirectory.isBlank() ? pathToRelativize.getFileName().toString() : Paths.get(extractedDirectory).relativize(pathToRelativize).toString();
            ;
```

Reply:
Updated